### ISSUE TITLE
Minor version (feature) is not detected as expected with Angular style

### ISSUE THREAD

**Issue by LoicViennois** [pawamoy/git-changelog#6]
**Tested version of git-changelog: `0.1.0`**

**Initial state:**

![image](https://user-images.githubusercontent.com/9892489/69341512-fdf4ce00-0c69-11ea-8a8a-f714961dfe42.png)

**Command:**
```
$ git-changelog -o CHANGELOG.md -s angular -t angular .
```

**Expected result:**
```
## v1.1.0

### Bug Fixes
- fix typo (632e64b)

### Features
- display python version (5fdf6a0)

## v1.0.0 (2019-11-19)
```

**Actual result:**
```
## v1.0.1

### Bug Fixes
- fix typo (632e64b)

### Features
- display python version (5fdf6a0)

## v1.0.0 (2019-11-19)
```

**Comment by LoicViennois** [comment]
Hello @pawamoy 
I propose a fix for this issue in this PR : https://github.com/pawamoy/git-changelog/pull/7
I took the opportunity to add two tests for angular style (feat and fix)

**Comment by pawamoy** [comment]
What a great report 😮  Thanks!

**Comment by LoicViennois** [comment]
no problem 👍

**Comment by LoicViennois** [comment]
@pawamoy do you plan to roll out a new version on PyPI any time soon ?

**Comment by pawamoy** [comment]
I do 🙂 I'll try to release it tonight.

**Comment by pawamoy** [comment]
Didn't get enough time, I'll finish and release this weekend :wink:

**Comment by pawamoy** [comment]
@LoicViennois quick question, with what Python version do you use git-changelog? I'm planning on dropping support for Python < 3.6.

**Comment by LoicViennois** [comment]
I use it with Python 3.7

**Comment by pawamoy** [comment]
I released version 0.2.0 yesterday. We'll close the issue if it works as expected.

**Comment by LoicViennois** [comment]
Great, thanks 👍
